backend/db/seeders/20230415035250-spot-seeder.js

```diff
@@ -29,7 +28,7 @@ module.exports = {
         city: "SomeOtherPlace",
         state: "CA",
         country: "USA",
-        lat: 24220.34242,
+        lat: 240.34242,
```

Comment:
min / max latitude : -90, 90

this passes since its only going through migrations and not the model file ( which is fine ).

Reply:
ahh yes, i tried to clean it up a bit and might have missed that. i do need to add some more validations as well for min and max